.github/workflows/32-bit-linux.yml

```diff
@@ -41,6 +41,7 @@ jobs:
           pip install cython numpy python-dateutil pytz pytest pytest-xdist pytest-asyncio>=0.17 hypothesis && \
           python setup.py build_ext -q -j2 && \
           python -m pip install --no-build-isolation --no-use-pep517 -e . && \
+          python -m pip list && \
```

Comment:
Can we remove this?

Reply:
Sure. I may add it back in a future PR as I think it may help for debugging purposed (other builds show their dependencies by default)

Reply:
Makes sense, thx for clarifying